.gitignore

```diff
@@ -161,4 +161,7 @@ cython_debug/
 
 **/env/
 **/key.txt
-**/local-auth.json
\ No newline at end of file
+**/local-auth.json
```

Comment:
dont forget to gcloudignore the same files

Reply:
Solved. We've updated the .gitignore file.

Reply:
@mikhailocampo About the error, we've forgot to update `features.dynamo.core` with the parent app module. We refactored that with the following code:
![image](https://github.com/radicalxdev/kai-ai-backend/assets/113047749/11333eb7-12ca-4558-81e0-a86026f3e5c2)

Reply:
It's working appropriately. Also, other people have tested the same PR and work for them. Here are screenshots from different local machines testing the PR:
![image](https://github.com/radicalxdev/kai-ai-backend/assets/113047749/a3cb4726-22d0-4112-93db-a753c0573c7a)
![image](https://github.com/radicalxdev/kai-ai-backend/assets/113047749/7a76669f-b7c2-48c5-93d9-0f7ab9eca663)
![image](https://github.com/radicalxdev/kai-ai-backend/assets/113047749/22a85969-fd3d-4bdc-8efa-382dc648b6ae)